src/languageserver/handlers/languageHandlers.ts

```diff
@@ -169,7 +185,17 @@ export class LanguageHandlers {
       return;
     }
 
-    return this.languageService.getFoldingRanges(textDocument, this.yamlSettings.capabilities.textDocument.foldingRange);
+    const capabilities = this.yamlSettings.capabilities.textDocument.foldingRange;
```

Comment:
There weren't any existing tests for `foldingRangeHandler` so I didn't add any to test this new functionality (just the underlying `getFoldingRanges`).